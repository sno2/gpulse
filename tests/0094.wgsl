@compute @workgroup_size(16,1,1)
fn main(@builtin(local_invocation_index) lid : u32) {
  for (var i = 0u; i < 10; i++) {
    workgroupBarrier();
    if (lid + i) > 7 {
      break;
    }
  }
}
