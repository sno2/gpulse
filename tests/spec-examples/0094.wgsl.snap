error: Expected expression, found 'for'.

  for (var i = 0u; i < 10; i++) {
  ~~~
error: Expected statement, found 'for'.

  for (var i = 0u; i < 10; i++) {
  ~~~
